### ISSUE TITLE
What is current state of translations in go?

### ISSUE THREAD

**Issue by VladislavsPerkanuks** [translate-agent/translate#4]

**Comment by VladislavsPerkanuks** [comment]
Somewhere at the end of 2019, the main maintainer **Marcel**, started to slow down with commits in the `golang.org/x/text`, with the last commit on Sep 15, 2021 ([ref](https://go-review.googlesource.com/q/project:text)).  Since that date, **Marcel** has stopped putting in contributions and reviewing, which has left a lot of new pull requests stuck without approval (since he was the one assigned).

For now `golang.org/x/text` has no main maintainer (as far as I understood) [ref](https://github.com/golang/go/issues/54481)

The bottom line is, the package will probably stay the same until a new maintainer steps up and makes changes. But there's no telling when that will happen if it ever does.
